### ISSUE TITLE
Feature Request: Past Review Details

### ISSUE THREAD

**Issue by kartikarora** [udacity/reviewer-dashboard-feedback#35]
https://simplydallas.github.io/udacityreviewparser/ used to give a lot of details about a review earlier. Could you add a similar functionality on the review list? It's useful in identifying plagiarism.

**Comment by kartikarora** [comment]
And feedback list in the new dash as well please!

**Comment by Digizard** [comment]
The student ratings area is part of #10, if that's what you mean?

**Comment by kartikarora** [comment]
@SittingFox Not just that. The tool I mentioned above used to give a lot of information about the review, basically everything that the api used to give out.

**Comment by Digizard** [comment]
Yeah, I just meant for the separate 2nd comment. I wasn't sure if you meant the ratings tab we have in the old dashboard. (Which I thought might not be the case, since that seems kind of like a random tag-on. But it might be a good opportunity to elaborate then.)

I also mentioned Dallas Frank's tool in one of my feature requests. I still like using it and think the builders of the new dashboard could definitely learn a lot from it. :smile:

**Comment by kartikarora** [comment]
True, that tool has been very useful. Not anymore though! :confused:

**Comment by kartikarora** [comment]
As for the second comment @SittingFox, I reckon that the old dash will be shutdown once the new one comes out of beta, hence the request for feedback list, in case they are not already working on it!

**Comment by Digizard** [comment]
Well I meant that, if you meant a separate issue, then maybe it should be a separate issue? But I guess #10 takes care of that in that case.

**Comment by kartikarora** [comment]
Ah yes, #10 does take care of it.
